<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>STOK_URUN_BIRIMLERI_BATCH</class>
 <widget class="QDialog" name="STOK_URUN_BIRIMLERI_BATCH">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>204</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>744</width>
    <height>204</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ÜRÜN BİRİMLERİ</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,50,1" columnstretch="50,1">
   <property name="sizeConstraint">
    <enum>QLayout::SetNoConstraint</enum>
   </property>
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,1,1,1">
     <item row="0" column="1">
      <widget class="QLabel" name="label_birim_adi">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Birim Adı</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_katsayi">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Katsayı</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_alt_birimi">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Alt Birimi</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="label_alis_fiyati">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Alış Fiyatı</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QLabel" name="label_satis_fiyati">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Satış Fiyatı</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_birim_adi_1">
       <property name="maxLength">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_birim_adi_2">
       <property name="maxLength">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_birim_adi_3">
       <property name="maxLength">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_birim_adi_4">
       <property name="maxLength">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QCommaEdit" name="commaEdit_katsayi_1">
       <property name="whatsThis">
        <string>Sayilari formatina(Or:1,000,234.50) donusturur. </string>
       </property>
       <property name="QCommaEdit" stdset="0">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QCommaEdit" name="commaEdit_katsayi_2">
       <property name="whatsThis">
        <string>Sayilari formatina(Or:1,000,234.50) donusturur. </string>
       </property>
       <property name="QCommaEdit" stdset="0">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QCommaEdit" name="commaEdit_katsayi_3">
       <property name="whatsThis">
        <string>Sayilari formatina(Or:1,000,234.50) donusturur. </string>
       </property>
       <property name="QCommaEdit" stdset="0">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QCommaEdit" name="commaEdit_katsayi_4">
       <property name="whatsThis">
        <string>Sayilari formatina(Or:1,000,234.50) donusturur. </string>
       </property>
       <property name="QCommaEdit" stdset="0">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QComboBox" name="comboBox_alt_birim_1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QComboBox" name="comboBox_alt_birim_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QComboBox" name="comboBox_alt_birim_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QComboBox" name="comboBox_alt_birim_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QCommaEdit" name="commaEdit_alis_fiyati_1">
       <property name="whatsThis">
        <string>Sayilari formatina(Or:1,000,234.50) donusturur. </string>
       </property>
       <property name="QCommaEdit" stdset="0">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QCommaEdit" name="commaEdit_alis_fiyati_2">
       <property name="whatsThis">
        <string>Sayilari formatina(Or:1,000,234.50) donusturur. </string>
       </property>
       <property name="QCommaEdit" stdset="0">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QCommaEdit" name="commaEdit_alis_fiyati_3">
       <property name="whatsThis">
        <string>Sayilari formatina(Or:1,000,234.50) donusturur. </string>
       </property>
       <property name="QCommaEdit" stdset="0">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
      </widget>
     </item>
     <item row="4" column="6">
      <widget class="QCommaEdit" name="commaEdit_alis_fiyati_4">
       <property name="whatsThis">
        <string>Sayilari formatina(Or:1,000,234.50) donusturur. </string>
       </property>
       <property name="QCommaEdit" stdset="0">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QCommaEdit" name="commaEdit_satis_fiyati_1">
       <property name="whatsThis">
        <string>Sayilari formatina(Or:1,000,234.50) donusturur. </string>
       </property>
       <property name="QCommaEdit" stdset="0">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QCommaEdit" name="commaEdit_satis_fiyati_2">
       <property name="whatsThis">
        <string>Sayilari formatina(Or:1,000,234.50) donusturur. </string>
       </property>
       <property name="QCommaEdit" stdset="0">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
      </widget>
     </item>
     <item row="3" column="7">
      <widget class="QCommaEdit" name="commaEdit_satis_fiyati_3">
       <property name="whatsThis">
        <string>Sayilari formatina(Or:1,000,234.50) donusturur. </string>
       </property>
       <property name="QCommaEdit" stdset="0">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
      </widget>
     </item>
     <item row="4" column="7">
      <widget class="QCommaEdit" name="commaEdit_satis_fiyati_4">
       <property name="whatsThis">
        <string>Sayilari formatina(Or:1,000,234.50) donusturur. </string>
       </property>
       <property name="QCommaEdit" stdset="0">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string> x </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string> x </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string> x </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string> x </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="5">
      <widget class="QLabel" name="label_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string> = </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QLabel" name="label_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string> = </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QLabel" name="label_7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string> = </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QLabel" name="label_8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string> = </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="8" colspan="3">
      <widget class="QLabel" name="label_urun_kartinda_goster">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Ürün Kartında Göster</string>
       </property>
      </widget>
     </item>
     <item row="1" column="9">
      <widget class="QCheckBox" name="checkBox_urun_kartinda_goster_1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="9">
      <widget class="QCheckBox" name="checkBox_urun_kartinda_goster_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="9">
      <widget class="QCheckBox" name="checkBox_urun_kartinda_goster_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="9">
      <widget class="QCheckBox" name="checkBox_urun_kartinda_goster_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>1. </string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_10">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>2. </string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_11">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>3. </string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_12">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>4. </string>
       </property>
      </widget>
     </item>
     <item row="4" column="11">
      <widget class="QPushButton" name="button_temizle_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Temizle</string>
       </property>
      </widget>
     </item>
     <item row="3" column="11">
      <widget class="QPushButton" name="button_temizle_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Temizle</string>
       </property>
      </widget>
     </item>
     <item row="2" column="11">
      <widget class="QPushButton" name="button_temizle_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Temizle</string>
       </property>
      </widget>
     </item>
     <item row="1" column="11">
      <widget class="QPushButton" name="button_temizle_1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Temizle</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QWidget" name="widget_batch_buttons" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCommaEdit</class>
   <extends>QLineEdit</extends>
   <header>QCommaEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEdit_birim_adi_1</tabstop>
  <tabstop>commaEdit_katsayi_1</tabstop>
  <tabstop>comboBox_alt_birim_1</tabstop>
  <tabstop>commaEdit_alis_fiyati_1</tabstop>
  <tabstop>commaEdit_satis_fiyati_1</tabstop>
  <tabstop>checkBox_urun_kartinda_goster_1</tabstop>
  <tabstop>button_temizle_1</tabstop>
  <tabstop>lineEdit_birim_adi_2</tabstop>
  <tabstop>commaEdit_katsayi_2</tabstop>
  <tabstop>comboBox_alt_birim_2</tabstop>
  <tabstop>commaEdit_alis_fiyati_2</tabstop>
  <tabstop>commaEdit_satis_fiyati_2</tabstop>
  <tabstop>checkBox_urun_kartinda_goster_2</tabstop>
  <tabstop>button_temizle_2</tabstop>
  <tabstop>lineEdit_birim_adi_3</tabstop>
  <tabstop>commaEdit_katsayi_3</tabstop>
  <tabstop>comboBox_alt_birim_3</tabstop>
  <tabstop>commaEdit_alis_fiyati_3</tabstop>
  <tabstop>commaEdit_satis_fiyati_3</tabstop>
  <tabstop>checkBox_urun_kartinda_goster_3</tabstop>
  <tabstop>button_temizle_3</tabstop>
  <tabstop>lineEdit_birim_adi_4</tabstop>
  <tabstop>commaEdit_katsayi_4</tabstop>
  <tabstop>comboBox_alt_birim_4</tabstop>
  <tabstop>commaEdit_alis_fiyati_4</tabstop>
  <tabstop>commaEdit_satis_fiyati_4</tabstop>
  <tabstop>checkBox_urun_kartinda_goster_4</tabstop>
  <tabstop>button_temizle_4</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
